.github/workflows/nvqc_regression_tests.yml

```
@@ -0,0 +1,351 @@
+name: Nvqc regression tests
+
+concurrency:
+  # only one integration tests workflow to be run at a time, since it involves pushing/cleaning up a test image (same tag).
+  group: ${{ github.workflow }}${{ github.event.workflow_run.name }}
+  cancel-in-progress: false
+
+# Run on request and every day at 3 AM UTC
+on:
+  workflow_dispatch:
+    inputs:
+      target:
+        description: 'Target (choose nightly to run like nightly tests)'
+        required: true
+        default: 'nightly'
+      cudaq_test_image:
+        type: string
+        required: false
+        default: 'nvcr.io/nvidia/nightly/cuda-quantum:latest' # If changed, update env defaults, too
+        description: 'CUDA Quantum image to run the tests in. Default to the latest CUDA Quantum nightly image'
+      commit_sha:
+        type: string
+        required: false
+        description: 'Commit SHA to pull the code (examples/tests) for testing. Default to the commit associated with the CUDA Quantum docker image if left blank'
+      workflow_id:
+        type: string
+        required: false
+        description: 'Workflow Id to retrieve the Python wheel for testing. Default to the wheels produced by the Publishing workflow associated with the latest nightly CUDA Quantum Docker image if left blank'
+      python_version:
+        type: choice
+        required: true
+        default: '3.10' # If changed, update env defaults, too
+        description: 'Python version to run wheel test'
+        options:
+        - '3.8'
+        - '3.9'
+        - '3.10'
+        - '3.11'
+
+  schedule:
+    - cron: 0 3 * * *
+
+env:
+  # NGC nv-quantum organization: pnyjrcojiblh
+  NGC_QUANTUM_ORG: pnyjrcojiblh
+  NGC_QUANTUM_TEAM: cuda-quantum
+  # <Backend>:<GPU Type>:<Instance Type>:<Min Instances>:<Max Instances>
+  NGC_NVQC_DEPLOYMENT_SPEC: NVCFInternal:A100:Standard_ND96amsr_A100_v4_1x:1:1
+  # If vars below are changed, it is recommended to also update the
+  # workflow_dispatch defaults above so they stay in sync.
+  cudaq_test_image: nvcr.io/nvidia/nightly/cuda-quantum:0.7.1
```

This value doesn't seem to be in sync with parameter default value above.
```suggestion
  cudaq_test_image: nvcr.io/nvidia/nightly/cuda-quantum:latest
```